### ISSUE TITLE
baelte command not found in my console

### ISSUE THREAD

**Issue by aresinodev** [kennethlarsen/baelte#10]
I have installed with "npm install -g baelte" and when I'm going to execute "baelte new project-name" the console say me "baelte is not recognized as an internal or external command,
program or batch file executable."
My SO is Windows.

help me, please

**Comment by kennethlarsen** [comment]
Hi @aaRonRJ 

Thank you for reporting this!

Sadly, I don't have a Windows machine to test this on. Did you get a success message after running `npm install -g baelte`?

What version of Node are you using?

**Comment by aresinodev** [comment]
Yes. Execute `npm install -g baelte` and when write `baelte new project-name` the console say me `baelte is not recognized as an internal or external command,
program or batch file executable.`.

Node version: 8.12.0
npm version: 6.4.1

**Comment by Bandit** [comment]
Are other `npm install -g` programs working after install? Might be a $PATH issue? Are you using WSL?

**Comment by fredguth** [comment]
I have the same problem o Mac OS. Other `npm install -g` have always worked.
